src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -47,23 +46,23 @@ While _written language_ and _character_ are slightly less precise, they should
 If you must use a word that your audience is unlikely to know, provide a definition using the [Key-term Aside component](/handbook/web-dev-components#asides).
 
 ## Make information easy to find
-Headings [shouldn't skip levels](/heading-levels). Also, don't add custom styles to headings—it makes the page hierarchy more difficult to follow for sighted readers.
+[Don't skip heading levels](/heading-levels). Don't add custom styles to headings—it makes the page hierarchy more difficult to follow.
 
 Spell out acronyms the first time they're used. For example: _Web Incubation Community Groups (WICG)_.
 
-Exceptions, edge cases, and other kinds of supplemental information should be right next to the primary content they're related to.
```

Comment:
This happens a lot. Switching between "X should be Y." instead of the imperative voice "Do x". Is that intentional or a mistake? I fixed where I saw it, but also wasn't sure if this was an intentional voice choice.

Reply:
Just an artifact of writing too quickly.

Reply:
Fixed in Google Doc